### ISSUE TITLE
can you tell me how to get ResNet152_v1d 

### ISSUE THREAD

**Issue by biBottle** [SJTU-Thinklab-Det/DOTA-DOAI#35]
hi,can you tell me how to get ResNet152_v1d besides google? 请问有国内源获取ResNet152_v1d的方式么

**Comment by yangxue0827** [comment]
https://github.com/yangxue0827/RotationDetection

**Comment by biBottle** [comment]
> https://github.com/yangxue0827/RotationDetection

Thx!!!  but then I have a new issue....can you help me deal with it?

NotFoundError (see above for traceback): Tensor name "resnet152_v1d/C1/conv0/BatchNorm/beta" not found in checkpoint files /home/u/下/DOTA-DOAI/FPN_Tensorflow_Rotation/data/pretrained_weights/resnet152_v1d.ckpt
         [[node save/RestoreV2 (defined at ../libs/networks/build_whole_network.py:702) ]]
![123](https://user-images.githubusercontent.com/22256390/121648803-7a5c3580-caca-11eb-8802-31b2e68d45c0.png)
